src/vm/stackwalk.cpp

```diff
@@ -1739,23 +1741,29 @@ StackWalkAction StackFrameIterator::Filter(void)
                             // frame matching the current frame, it means that the funclet stack frame was reclaimed.
                             StackFrame sfFuncletParent;
                             ExceptionTracker* pCurrTracker = pTracker;
-                            bool hasFuncletStarted = m_crawl.pThread->GetExceptionState()->GetCurrentEHClauseInfo()->IsManagedCodeEntered();
+
+                            bool hasFuncletStarted = pTracker->GetEHClauseInfo()->IsManagedCodeEntered();
 
                             while (pCurrTracker != NULL)
                             {
-                                if (hasFuncletStarted)
+                                // Ignore exception trackers for filter clauses, since their frames are not on the stack on Windows either
```

Comment:
@janvorli Is this comment correct?  It seems to disagree with your new comment in 1732